core/state/database.go

```diff
@@ -281,6 +273,9 @@ func (db *cachingDB) OpenStorageTrie(stateRoot common.Hash, address common.Addre
 }
 
 func (db *cachingDB) CacheAccount(root common.Hash, t Trie) {
+	if db.TrieDB().Scheme() == rawdb.PathScheme {
```

Comment:
Could you add a `TODO` or some comments here for why disable cache account in path scheme?

Reply:
yes